frontend/src/api/index.jsx

```diff
@@ -0,0 +1,57 @@
+const HOST = "http://localhost:3001";
```

Comment:
Did you get this to ping the backend? Flask is listening on port 5000 and the correct pathway would be:
`http://localhost:5000/api/auth/desiredendpoint`

Reply:
Okay, I will fix it